cpp/models/abm/household.cpp

```diff
@@ -58,6 +58,9 @@ void add_household_to_world(World& world, const Household& household)
 {
     auto home    = world.add_location(LocationType::Home);
     auto members = household.get_members();
+    world.get_individualized_location(home).set_capacity(household.get_total_number_of_members(),
+                                                         household.get_total_number_of_members() *
+                                                             household.get_space_per_member());
```

Comment:
This is okay for now, but please add an issue where we specify how to enhance the world creation with households and the adder functions.